mrchecker-framework-modules/mrchecker-database-module/src/main/java/com/capgemini/mrchecker/database/core/base/runtime/RuntimeParameters.java

```diff
@@ -0,0 +1,51 @@
+package com.capgemini.mrchecker.database.core.base.runtime;
+
+import com.capgemini.mrchecker.test.core.base.runtime.RuntimeParametersI;
+
+public enum RuntimeParameters implements RuntimeParametersI {
+
+	DATABASE_TYPE("databaseType", ""),
+	JPA_PROVIDER("provider", "hibernate"),
+	DATABASE_USERNAME("databaseUsername", "");
+
+	private String paramName;
+	private String paramValue;
+	private String defaultValue;
+
+	private RuntimeParameters(String paramName, String defaultValue) {
+		this.paramName = paramName;
+		this.defaultValue = defaultValue;
+		setValue();
+	}
+
+	@Override
+	public String getValue() {
+		return this.paramValue;
+	}
+
+	@Override
+	public String toString() {
+		return paramName + "=" + this.getValue();
+	}
+
+	@Override
+	public void refreshParameterValue() {
+		setValue();
+	}
+
+	private void setValue() {
+
+		String paramValue = System.getProperty(this.paramName);
+		paramValue = isSystemParameterEmpty(paramValue) ? this.defaultValue : paramValue.toLowerCase();
```

Comment:
Shouldn't we enforce `defaultValue` to be also lower-case?

Reply:
I left this to fulfill future requirements. This is base for all types of modules inside MrChecker. Syntax is unified for all modules.

Reply:
Discussed